common/scripts/configs/master_integrations.sql

```diff
@@ -340,6 +340,12 @@ do $$
       values ('577de63321333398d11a1123', 93, 'ircCreds', 'IRC Credentials', 'generic', false, 'generic', 5012, '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2018-09-11', '2018-09-11');
     end if;
 
+    -- airBrakeKey master integration
+    if not exists (select 1 from "masterIntegrations" where "name" = 'airBrakeKey' and "typeCode" = 5012) then
+      insert into "masterIntegrations" ("id", "masterIntegrationId", "name", "displayName", "type", "isEnabled", "level", "typeCode", "createdBy", "updatedBy", "createdAt", "updatedAt")
+      values ('577de63321333398d11a1124', 94, 'airBrakeKey', 'AirBrake', 'generic', true, 'generic', 5012, '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2018-09-11', '2018-09-11');
```

Comment:
are we sure that `isEnabled` should be true? In the previous integrations, the `isEnabled` field is false.

Reply:
Yes because the previous integrations are
- bitbucketServerBasicAuth
- gerritBasic
- gerritBasicAuth
- ircCreds

They (mostly scm addons and some notification addons) are supposed to be enabled manually by the Admin from Admiral, hence they are disabled by default. But in case of other integrations like jira, hipchat, slack, azurekeys etc. (integrations that doesn't any configuration by admin), we enable it by default.

![image](https://user-images.githubusercontent.com/4211715/53152394-70515880-35db-11e9-97de-8c663f1d7f29.png)